### PR TITLE
✨ admin page: add schedules

### LINKED ISSUE #318: admin: data management: planning form

### PR DESCRIPTION
**Pull request by bertcappelle**
Closes #318 

I'm not that sure about the naming of the 2 forms, suggestions are welcome

**Afval form:**
This form is made to fill in which waste type should be put outside/inside. 

- Clicking the logos changes the state, there are 3 states (nothing, inside and outside)
- Bulk action: clicking a table column header changes all the states of the column
- The weekcalendar can also be duplicated to the following weeks (it's set to a maximum of 100 days right now). This will first delete all the waste entries in those following weeks and then copy the calendar entries. This can take long. When it's set to 0, it only changes the selected week.
- In the dropdown menu you can select which tours you want to change

image

**Planningen form:**
This form is for assigning students to a schedule at a specific time. The calendar is also shown so you can see what is planned that day with the selected day highlighted. It is not editable.

image

### CODE REVIEW

**Comment by milachae** [comment]
@bertcappelle ,

It already looks really good! but I found some odd things:
- It is not really clear that you need first need to add the afval schedule before adding a real schedule. Is it possible to just say that somewhere?
- If it is easily doable, it can be handy to automatically set the "afval ophalen" to the next day. I think it could also be possible that "afval" needs to be put outside on sunday and back inside on monday. I think that this form is not compatible with this situation. Maybe just add the previous sunday or next monday to the "afval" tab.
- When no buildings are present for a specific route, the "afval" tab behaves a little bit weird, same with the "planningen" tab, maybe just put "Geen gebouwen voor deze route" if no buildings are present.

File: frontend/src/components/forms/ScheduleForm.jsx [hunk 1]
```diff
@@ -1,10 +1,208 @@
...
+      router.reload();
+    } catch (e) {
+      alert(e);
+    }
+  };
+
+  const onDelete = async () => {
+    try {
+      await scheduleService.deleteById(id);
+      await router.push(`/admin/data_toevoegen/planningen`);
```
Comment (milachae) [inline frontend/src/components/forms/ScheduleForm.jsx:54]: `admin` has been change to `beheer`

**Review by milachae — DISMISSED** [review]

**Comment by bertcappelle** [comment]
> @bertcappelle ,
> 
> It already looks really good! but I found some odd things:
> 
> * It is not really clear that you need  to first add the "afval" schedule before adding a real schedule. Is it possible to just say that somewhere?
> * If it is easily doable, it can be handy to automatically set the "afval ophalen" to the next day.
> * I think it could also be possible that "afval" needs to be put outside on sunday and back inside on monday. I think that this form is not compatible with this situation. Maybe just add the previous sunday or next monday to the "afval" tab.
> * When no buildings are present for a specific route, the "afval" tab behaves a little bit weird, same with the "planningen" tab, maybe just put "Geen gebouwen voor deze route" if no buildings are present.

All these changed should be implemented now.

File: frontend/src/components/forms/ScheduleForm.jsx (hunk 1 above)
Comment (bertcappelle) [reply frontend/src/components/forms/ScheduleForm.jsx]: This is fixed now

**Review by renevds — APPROVED** [review]

**Pull request merged**